packages/react/src/use-render/useRender.ts

```diff
@@ -64,7 +64,13 @@ export namespace useRender {
      * internal ones.
      */
     props?: Record<string, unknown>;
+    /**
+     * If `false`, the hook will skip most of its internal logic and return `null`.
+     * This is useful for rendering a component conditionally.
+     * @default true
+     */
+    enabled?: boolean;
```

Comment:
I'm not sure if `shouldRender` wouldn't be better? More clear & semantic, and matches the variable that we're going to assign to this prop most of the time.

Reply:
I went with `enabled` as it's commonly used in libraries (Floating UI, TanStack, etc.) for this purpose.